autoload/lsp/internal/document_hover/under_cursor.vim

```diff
@@ -74,6 +74,13 @@ function! lsp#internal#document_hover#under_cursor#do(options) abort
         \ )
 endfunction
 
+function! lsp#internal#document_hover#under_cursor#getpreviewwinid() abort
```

Comment:
can we have an alias to this function as in `lsp#document_hover_preview_winid()` that calls `lsp#internal#document_hover#under_cursor#getpreviewwinid()` ? I would like to avoid users using `lsp#interal`.